acl_loader/main.py

```diff
@@ -642,9 +692,10 @@ def deny_rule(self, table_name):
             rule_props["ETHER_TYPE"] = str(self.ethertype_map["ETHERTYPE_IPV4"])
         return rule_data
 
-    def convert_rules(self):
+    def convert_rules(self, is_time_based):
```

Comment:
Same previous comment, we'd better use enum type here.
We can do refactor in future. Add this comment here as a reminder

Reply:
Noted.